plugins/destination/plugin_testing_overwrite_delete_stale.go

```diff
@@ -72,13 +72,11 @@ func (*PluginTestSuite) destinationPluginTestWriteOverwriteDeleteStale(ctx conte
 
 	secondSyncTime := syncTime.Add(time.Second).UTC()
 
-	// copy first resource but update the sync time
-	updatedResource := schema.DestinationResource{
-		TableName: table.Name,
-		Data:      make(schema.CQTypes, len(resources[0].Data)),
+	updatedResource := createTestResources(table, sourceName, secondSyncTime, 1)[0]
+	for _, colIndex := range []int{2, 3, 7} {
+		old := resources[0].Data[colIndex].Get()
+		return updatedResource.Data[colIndex].Set(old)
```

Comment:
you exit early, so only `colIndex=2` is processed

Reply:
fixed